.jenkins/Jenkinsfile

```diff
@@ -16,7 +16,25 @@ def ACCTest(String label, String compiler, String unit, String suite) {
             checkout scm
 
             timeout(15) {
-                sh "./scripts/test-build-config -p $unit -b $suite --compiler=$compiler --disable_sim"
+                dir('build') {
+                    sh """
+                    if [ "${compiler}" == "gcc" ]; then
+                        export CC="gcc"
+                        export CXX="g++"
+                    elif [ "${compiler}" == "clang-7" ]; then
+                        export CC="clang-7"
+                        export CXX="clang++-7"
+                    fi
+
+                    if [ "${unit}" == "SGX1FLC" ]; then
+                        cmake .. -DCMAKE_BUILD_TYPE=${suite} -DUSE_LIBSGX=1
```

Comment:
`-DUSE_LIBSGX=ON` is the default, so this could would always turn it on.

Reply:
(Also prefer `ON/OFF` for CMake instead of `1/0`. They're equivalent but the former reads a bit more clearly.)

Reply:
Duly noted for ON/OFF instead of 1/0, I agree it reads a whole lot better.

So I should just remove `-DUSE_LIBSGX=1` from line 30?

Reply:
Yes, but in any tests that are explicitly not supposed to link (so, simulator mode) would now need `-DUSE_LIBSGX=OFF`.